File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Farinha/03.pl
#Programa que verifica se um número fornecido pelo utilizador é positivo, negativo ou nulo.
print "Digite um numero:\n";
$numero = <STDIN>;
if ($numero > 0)
{
	print "Positivo\n";
}
elsif ($numero < 0)
{
	print "Negativo\n";
}
else
{
	print "Nulo\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Farinha/03.rb
#Programa que verifica se um número fornecido pelo utilizador é positivo, negativo ou nulo.
puts "Digite um numero:"
numero = gets
if numero.to_i > 0
  puts "Positivo"
elsif numero.to_i < 0
  puts "Negativo\n"
else
  puts "Nulo"
end